pandas/tseries/offsets.py

```diff
@@ -163,6 +164,7 @@ def __add__(date):
     normalize = False
 
     def __init__(self, n=1, normalize=False, **kwds):
+        assert n == int(n)
```

Comment:
huh? why are you putting asserts here

Reply:
for this case I actually do like 
``self.n = self._validate_n(n)``

which would raise on a non-integer convertible passed in (with a helpful message that includes the class name); that's the reason to make this a method)

Reply:
I'll get rid of the asserts here, handle them later.